<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeCommands</class>
 <widget class="QDialog" name="ChangeCommands">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR CARTER</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Configure Your Own Commands</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>填写格式示例：aa,AA,0xaa,0xAA</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>160</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit_FORWARD"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_LEFT"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_RIGHT"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_LEFT_FORWARD"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_RIGHT_FORWARD"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_STOP"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>449</y>
     <width>235</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_Save">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Cancle">
      <property name="text">
       <string>Cancle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>某一项不填写则该项配置保持不变</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>160</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_FORWARD">
      <property name="text">
       <string>FORWARD</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_LEFT">
      <property name="text">
       <string>LEFT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_RIGHT">
      <property name="text">
       <string>RIGHT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_LEFT_FORWARD">
      <property name="text">
       <string>LEFT_FORWARD</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_RIGHT_FORWARD">
      <property name="text">
       <string>RIGHT_FORWARD</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_STOP">
      <property name="text">
       <string>STOP</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
